README.md

```diff
@@ -65,14 +71,13 @@ Please see our [user manual](docs/user_manual.md) for instructions on how to use
 
 ##### Run dashboard
 
-Run the command below and and click (or ctrl/cmd+click) on the link shown.
+Run the command below and and open the resulting link in a browser (often something like `http://127.0.0.1:8050/`).
+Note that the dashboard has only been tested on Chrome and Firefox.
```

Comment:
I also tested it on Safari, and it works fine, but we should only add that if we keep on testing it there.